src/index.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
Since u use inline style is it necessary to use a css file for only background color?

Reply:
it is the same background everywhere, I think it will be populated with more stuff later

Reply:
changed to be in App.jsx, index.css removed